README.md

```diff
@@ -24,6 +24,31 @@ videojs('player_id').ready(function () {
 
       certificateUrl: '/path/to/certificate',
       licenseUrl: '/path/to/license',
+
+      // Optional settings
+
+      // Instead of a certificateUrl send the cert in a string
+      certificate: 'base64string',
+
+      // Object to add any extra license request headers that might be needed
+      licenseRequestHeaders: {
+        'my-extra-header': 'some important information'
+      },
+
+      // Function to call back with license response content if it errors
+      licenseResponseErrorContent: function (content) {
```

Comment:
see some of the comments below, ideally I would like to consolidate naming to one specific style.

shooting from the hip;

- `onLicenseLoad` (can be changed if needed)
- `onLicenseError({ error })`
- `onLicenseSuccess({ headers })`

signatures would be able to be grouped and are conventionally named, they can be enhanced with data as needed.